cypress/integration/testBorna/test.cy.js

```diff
@@ -0,0 +1,179 @@
+import uniqid from "uniqid";
+
+function getPlaceholder(placeholder) {
+    return cy.get('input[placeholder="' + placeholder + '"]');
+}
+
+function makeid(length) {
+    var result = "";
+    var characters = "abcdefghijklmnopqrstuvwxyz";
+    var charactersLength = characters.length;
+
+    for (var i = 0; i < length; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+}
+
+context("Blocks Page", () => {
+    var import_url =
```

Comment:
Is there a reason why this is hardcoded here? This looks like it shouldn't be here.

Reply:
This was from when I was writing tests for importing blocks. So I exported the necessary files and then hardcoded the links for importing in order to test those in the future. This however is definitely not a permanent solution and it would be much better if I got a new one each testing round.